### DISCUSSION TITLE
Downgrading question on verified mode?

### DISCUSSION THREAD

**Discussion by FunnyCorgi** [3kh0/ext-remover#1075] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
Do I need to disable verified mode to downgrade to 105?

**Comment by Averagedude69420** [comment]
It's not possible to downgrade to v105 if your Chromebook has received the v114 update, most Chromebooks can only downgrade to v112 or v111

**Reply by FunnyCorgi** [reply]
If I downgrade to 111 or 112, can I then go to 105?

**Reply by LizzyEll** [reply]
No, the update kinda c*** blocks you. Even if you downgrade to another version, you can't go down from there.

**Reply by FunnyCorgi** [reply]
What if my chromebook was on 118 and newly enrolled when I got it?

**Comment by Averagedude69420** [comment]
> Do I need to disable verified mode to downgrade to 105?
what do you even mean by disable verified mode?

**Reply by FunnyCorgi** [reply]
Like press control + d on the first screen, and then do enter.

**Comment by S-PScripts** [comment]
Closing unless funnycorgi still needs an answer

**Reply by S-PScripts** [reply]
if you want you can try downgrading with verified mode  on first and if that doesn't work, try it without

**Comment by FunnyCorgi** [comment]
I'm good now, this can be closed.
